gormsupport/cleaner/db_clean.go

```diff
@@ -47,6 +48,10 @@ func DeleteCreatedEntities(db *gorm.DB) func() {
 		key     interface{}
 	}
 	var entires []entity
+	hookRegistered := db.Callback().Create().Get(hookName) != nil
+	if hookRegistered {
+		hookName += "-" + uuid.NewV4().String()
+	}
 	db.Callback().Create().After("gorm:create").Register(hookName, func(scope *gorm.Scope) {
 		logrus.Debug(fmt.Sprintf("Inserted entities from %s with %s=%v", scope.TableName(), scope.PrimaryKey(), scope.PrimaryKeyValue()))
```

Comment:
@aslakknutsen @kwk git blame says something different, so it wasn't me. Anyway, let's use `log`